source/config/__init__.py

```diff
@@ -1283,6 +1284,22 @@ def __setitem__(
 		self.manager._markWriteProfileDirty()
 		self._cache[key] = val
 
+		# Alias [documentFormatting][reportFontAttributes] for backwards compatibility.
+		# Todo: Remove in 2025.1.
+		if self.path == ("documentFormatting",):
```

Comment:
could you add a version check here like suggested in this doc: https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md

Reply:
could you also move this into a helper function